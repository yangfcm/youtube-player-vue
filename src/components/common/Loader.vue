<template>
  <div>
    <app-blank></app-blank>
    <div class="ui active centered inline loader"></div>
    <app-blank></app-blank>
  </div>
</template>

<script>
import Blank from "./Blank";
export default {
  components: {
    appBlank: Blank,
  },
};
</script>

<style scoped></style>
