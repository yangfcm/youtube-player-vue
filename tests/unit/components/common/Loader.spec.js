import { shallowMount } from "@vue/test-utils";
import Loader from "@/components/common/Loader";

test("Loader component can render correctly", () => {
  const wrapper = shallowMount(Loader);
  expect(wrapper).toMatchSnapshot();
});
