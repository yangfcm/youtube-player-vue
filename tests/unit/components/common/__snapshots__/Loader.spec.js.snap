// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component can render correctly 1`] = `
<div>
  <app-blank-stub></app-blank-stub>
  <div class="ui active centered inline loader"></div>
  <app-blank-stub></app-blank-stub>
</div>
`;
